### DISCUSSION TITLE
[Projects Beta] Add Note to Project

### DISCUSSION THREAD

**Discussion by kob123** [community/community#7544] [Projects and Issues]
Creating multiple views (especially the new table view) of the Projects Beta is great, but the biggest thing missing imo is the ability to add a **note**.

Just like you can in the [existing non-beta Projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/adding-notes-to-a-project-board).

Surprised I can't find this on the [roadmap](https://github.com/github/roadmap/projects/1).

This has been very helpful in non-beta Projects for creating a card that can live in one column that doesn't need to be a new issue, but can reference the existing issue.

Or, for creating an explanatory note about the column itself, which will be helpful once we can create Board view columns based on any custom field, [which thankfully I do see on the roadmap.](https://github.com/github/roadmap/issues/282)

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/4422
